api/src/main/java/io/opentelemetry/trace/SpanId.java

```diff
@@ -107,6 +107,21 @@ public static SpanId fromBytes(byte[] src, int srcOffset) {
     return new SpanId(BigendianEncoding.longFromByteArray(src, srcOffset));
   }
 
+  /**
+   * Transforms a byte[] representation of a span id (eg. from a protobuf) and returns a base 16
+   * representation of the same.
+   *
+   * @param bytes The byte array containing the span id.
+   * @return A base 16 representation of the span id.
+   * @since 0.1.0
+   */
+  public static String asLowerBase16(byte[] bytes) {
```

Comment:
There is a `public String toLowerBase16()`;

Reply:
There is, but that requires us to create a `SpanId` instance in order to use it. This static method would save us an object allocation during export.

Reply:
Ok, that is fair. But if we go with exposing a SpanData or getters on ReadableSpan this is not needed correct? Should we focus on resolving that issue first?

Reply:
Is there time for that work before the alpha, or should we plan on that for post-alpha?

Reply:
I would encourage to do it before alpha.

Reply:
Cool. If we have a plan in place (like which objects we want to pass over to the exporters), we'd be happy to take a swing at implementing this. 

Have we settled on a plan?  `SpanData`? `ReadableSpan` plus more getters? Something else?

Reply:
I would go with a SpanData approach if nobody has any objections, some advantages:
1. We can ensure immutability of the data.
1. We can ensure that all the referenced object during lazy events are no longer referenced by not keeping a reference to the ReadableSpan.
1. If multiple exporters the conversion happens only once to the SpanData.

Reply:
cool. We'll (New Relic folks) start figuring this out tomorrow, first thing. Thanks for your thoughts!